pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -1046,6 +1046,58 @@ def test_td64arr_add_sub_numeric_arr_invalid(self, box_with_array, vec, dtype):
         with pytest.raises(TypeError):
             vector - tdser
 
+    # TODO: box and de-duplicate
```

Comment:
```suggestion
    # TODO: parameterize over box and de-duplicate
```

Is that what you mean?